tests/start-cloud.sh

```
@@ -0,0 +1,13 @@
+docker rm -f $(docker ps --format '{{.ID}}.{{.Names}}' -a | grep 'cloud'| sed 's/\..*$//')
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)